common/main/es_AR.xml

```diff
@@ -46,7 +46,7 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 						<dateFormatItem id="yyyyM" draft="contributed">M-y G</dateFormatItem>
 					</availableFormats>
 					<intervalFormats>
-						<intervalFormatFallback draft="unconfirmed">{0} a el {1}</intervalFormatFallback>
+						<intervalFormatFallback draft="unconfirmed">{0} 'a el' {1}</intervalFormatFallback>
```

Comment:
These data changes seem good, but I am surprised that we need them in some of the fairly visible locales. Have you checked that ICU (without these changes) indeed writes bad output here?